### PR TITLE
Change log update

### PR DESCRIPTION
**Pull request by akshita31**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,6 +15,7 @@
 
 * Added support to configure the default *.sln file loaded when opening a project with multiple *.sln files in the root. _(Contributed by [@janaka](https://github.com/janaka))_ (PR: [#2053](https://github.com/OmniSharp/omnisharp-vscode/pull/2053)) 
 
+* Added support for tracking opening, closing and changing of virtual documents that don't exist on disk(PR: [#2436] (https://github.com/OmniSharp/omnisharp-vscode/pull/2436)) _(Contributed by [@NTaylorMullen](https://github.com/NTaylorMullen))
```
Comment (DustinCampbell) [inline CHANGELOG.md]: add a period after `disk` and a space before `(PR:`. Should be `disk. (PR:`

File: CHANGELOG.md [hunk 2]
```diff
@@ -29,6 +30,10 @@
 * Added test execution output to the output of the Run/Debug Test CodeLens. (PR: [#2337](https://github.com/OmniSharp/omnisharp-vscode/pull/2337), [#2343](https://github.com/OmniSharp/omnisharp-vscode/pull/2343), [omnisharp-roslyn#1203](https://github.com/OmniSharp/omnisharp-roslyn/pull/1203))
 * Fixed a bug where a debug session could not be started and duplicate logs were displayed after a previous one failed due to build failure. (PR: [#2405](https://github.com/OmniSharp/omnisharp-vscode/pull/2405), [omnisharp-roslyn#1239](https://github.com/OmniSharp/omnisharp-roslyn/pull/1239)) 
 
+#### Options
+
+* Added `monoPath` option to use the mono installation at the specified path when the `useGlobalMono` is set to "always" or "auto".(PR: [#2425] (https://github.com/OmniSharp/omnisharp-vscode/pull/2425)) _(Contributed by [@shana](https://github.com/shana))
```
Comment (DustinCampbell) [inline CHANGELOG.md]: Same feedback. Add a space before `(PR:`

File: CHANGELOG.md (hunk 1 above)
Comment (DustinCampbell) [reply CHANGELOG.md]: Also, remove the space between `[#2436]` and `(https://`. Otherwise, your markdown link is broken.

File: CHANGELOG.md (hunk 2 above)
Comment (DustinCampbell) [reply CHANGELOG.md]: Also, remove the space between `[#2425]` and `(https://`. Otherwise, your markdown link is broken.

File: CHANGELOG.md [hunk 3]
```diff
@@ -15,6 +15,7 @@
 
 * Added support to configure the default *.sln file loaded when opening a project with multiple *.sln files in the root. _(Contributed by [@janaka](https://github.com/janaka))_ (PR: [#2053](https://github.com/OmniSharp/omnisharp-vscode/pull/2053)) 
 
+* Added support for tracking opening, closing and changing of virtual documents that don't exist on disk. (PR: [#2436](https://github.com/OmniSharp/omnisharp-vscode/pull/2436)) _(Contributed by [@NTaylorMullen](https://github.com/NTaylorMullen))
```
Comment (rchande) [inline CHANGELOG.md]: Missing trailing underscore

File: CHANGELOG.md [hunk 4]
```diff
@@ -29,6 +30,10 @@
 * Added test execution output to the output of the Run/Debug Test CodeLens. (PR: [#2337](https://github.com/OmniSharp/omnisharp-vscode/pull/2337), [#2343](https://github.com/OmniSharp/omnisharp-vscode/pull/2343), [omnisharp-roslyn#1203](https://github.com/OmniSharp/omnisharp-roslyn/pull/1203))
 * Fixed a bug where a debug session could not be started and duplicate logs were displayed after a previous one failed due to build failure. (PR: [#2405](https://github.com/OmniSharp/omnisharp-vscode/pull/2405), [omnisharp-roslyn#1239](https://github.com/OmniSharp/omnisharp-roslyn/pull/1239)) 
 
+#### Options
+
+* Added `monoPath` option to use the mono installation at the specified path when the `useGlobalMono` is set to "always" or "auto". (PR: [#2425](https://github.com/OmniSharp/omnisharp-vscode/pull/2425)) _(Contributed by [@shana](https://github.com/shana))
```
Comment (rchande) [inline CHANGELOG.md]: Same

**Comment by akshita31** [comment]
@DustinCampbell The integration tests are failing here on the Codelens test. Any idea why that might be happening ?

**Comment by akshita31** [comment]
@DustinCampbell Can you look into why the integration test for the codelens provider is failing ?

**Pull request merged**